modules/doc/content/getting_started/installation/conda.md

```diff
@@ -1,86 +1,23 @@
 # Conda MOOSE Environment
 
-Our preferred method for obtaining libraries necessary for MOOSE based Application development, is via Conda's myriad array of libraries. Follow these instructions to create an environment on your machine using Conda. At this time, Windows is not supported. Effort is being made towards this direction. But it is some time out.
+Our preferred method for obtaining libraries necessary for MOOSE based
+Application development, is via Conda's myriad array of libraries. Follow these
+instructions to create an environment on your machine using Conda. At this time,
+Windows is not supported. Effort is being made towards this direction. But it is
```

Comment:
very minor, but while I'm looking at this, maybe Link to Windows directions, and comma between the two sentences?

Reply:
I agree with the comma separation. My grammar skills have always been found wanting.